docs/recipes/tracking-page-view.md

```diff
@@ -0,0 +1,28 @@
+Tracking Page View
+========================
+
+If you need to track a page view, use [request.onComplete](../brisket.requestobject.md#requestoncompletefn), and [request properties](../brisket.requestobject.md#whats-in-the-request-object):
+
+```js
+var BookRouter = Brisket.RouterBrewery.create({
+
+  routes: {
+    'books/:bookId': 'books'
+  },
+
+  books: function(bookId, layout, request, response) {
+
+    request.onComplete(function() {
+      ga("set", "referrer", request.referrer);
```

Comment:
For the completeness of the example, could you `require` in ga from somewhere so that the reader does not become confused?

for example (at the top)

```js 
var ga = require("google-analytics");
```